### DISCUSSION TITLE
Why last release version is 9.10.0 but in SolutionInfo.cs in sourcecode still 9.9.1

### DISCUSSION THREAD

**Discussion by thienvc** [dnnsoftware/Dnn.Platform#4756] [General]
`https://github.com/dnnsoftware/Dnn.Platform/blob/f740efc57980100f451a78be85a86f090c7b0882/SolutionInfo.cs#L16-L18`
[https://github.com/dnnsoftware/Dnn.Platform/blob/f740efc57980100f451a78be85a86f090c7b0882/SolutionInfo.cs#L16-L18](url)

**Comment by bdukes** [comment]
This is covered by #4753 (fixed by PR #4754, yet unmerged). Thanks!
